includes/class-database-cleanup.php

```
@@ -19,634 +19,636 @@
+					$result = self::invoke_cleanup_method( $method );
+				}
 
-			if ( is_wp_error( $result ) ) {
-				// Translators: %s is the cleanup method name that failed.
-				new Log( sprintf( __( 'Auto cleanup failed: %s', 'performance-optimisation' ), $method ) );
+				if ( is_wp_error( $result ) ) {
+					// Translators: %s is the cleanup method name that failed.
+					new Log( sprintf( __( 'Auto cleanup failed: %s', 'performance-optimisation' ), $method ) );
+				}
 			}
 		}
```

PHPMD (2.15.0)

[error] The method auto_clean is not named in camelCase. (undefined)

(CamelCaseMethodName)